aws/resource_aws_autoscaling_group_test.go

```diff
@@ -3557,10 +3600,12 @@ resource "aws_autoscaling_group" "test" {
       }
 
       override {
-        instance_type = "t2.micro"
+        instance_type   = "t2.micro"
+        weighted_capacity = "1"
```

Comment:
One quick question -- was this change the existing test configurations intentional? The new test below seems to capture the new behavior and we prefer to leave existing test configurations to check for regressions. If we want, it may be best to instead verify the existing configuration without this new argument got empty string (`""`) values in the Terraform state.

Reply:
Thanks for reviewing this @bflad!

Please note that `TestAccAWSAutoScalingGroup_MixedInstancesPolicy_LaunchTemplate_Override_InstanceType` is already testing this scenario (no weighted_capacity).

Having said that, I'm happy to go back to the original version of `TestAccAWSAutoScalingGroup_MixedInstancesPolicy` if you prefer.